lib/Echidna/Config.hs

```diff
@@ -59,14 +60,16 @@ instance FromJSON EConfig where
                           <*> v .:? "seqLen"      .!= 100
                           <*> v .:? "shrinkLimit" .!= 5000
                           <*> pure Nothing
+                          <*> v .:? "seed"
         names = const $ const mempty :: Names
-        ppc = cc <&> \c x -> runReader (ppCampaign x) (c, names)
-        style :: Y.Parser (Campaign -> String)
+        ppc = cc <&> \c x _ -> runReader (ppCampaign x) (c, names)
+        style :: Y.Parser (Campaign -> Maybe Int -> String)
         style = v .:? "format" >>= \case (Nothing :: Maybe String) -> ppc
                                          (Just "text")             -> ppc
-                                         (Just "json")             -> pure $ unpack . encode
-                                         (Just "none")             -> pure $ const ""
-                                         _                         -> M.fail
+                                         (Just "json")             -> pure . flip $ \g ->
```

Comment:
Nit: we shouldn't need to `flip` a lambda function, and I think that makes this code less clear

Reply:
Changing the order of the arguments of `_finished` will avoid the `flip`, but the `flip` here makes the code more readable as we will avoid an application of the massive chain of functions to the second parameter.